utils/symbol.c

```diff
@@ -1430,3 +1423,71 @@ void print_symtabs(struct symtabs *symtabs)
 		symbol_putname(sym, name);
 	}
 }
+
+static unsigned long kaddr;
```

Comment:
Also getter/setter functions should be renamed.  Like `get_kernel_base()` and `set_kernel_base()`.